extensions/youtube-music-desktop-app-controller/package.json

```diff
@@ -0,0 +1,145 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "youtube-music-desktop-app-controller",
+  "title": "Youtube Music Desktop App Controller",
+  "description": "Controle Youtube Music Desktop App from Raycast",
+  "icon": "extension-icon.png",
+  "author": "muditmahajan21",
+  "license": "MIT",
+  "categories": ["Media", "Applications"],
+  "commands": [
+    {
+      "name": "ytmd",
+      "title": "YTMdesktop Setup",
+      "description": "Control Youtube Music Desktop App from Raycast",
+      "mode": "view"
+    },
+    {
+      "name": "playPause",
+      "title": "Play/Pause",
+      "subtitle": "Play/Pause for Youtube Music Desktop App",
+      "description": "Play or pause the current track in Youtube Music Desktop App",
+      "mode": "no-view"
+    },
+    {
+      "name": "play",
+      "title": "Play",
+      "subtitle": "Play for Youtube Music Desktop App",
```

Comment:
This should show the name of the extension, it's not for describing - the same goes for all commands 🙂

Reply:
Updated in https://github.com/raycast/extensions/pull/14379/commits/91fad14bf6a729551bf62f7b6826997bdbf8c9ee